library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/MediaRelaySetsRelBrandDoctrineRepository.php

```diff
@@ -30,4 +31,40 @@ public function __construct(
             $entityPersisterInterface
         );
     }
+
+    /** @return int[] */
+    public function getMediaRelaySetIdsByBrandAdmin(AdministratorInterface $admin): array
+    {
+        if (!$admin->isBrandAdmin()) {
+            throw new \DomainException('User must be brand admin');
+        }
+
+        $brandId = $admin->getBrand()?->getId();
+        if (is_null($brandId)) {
+            return [];
+        }
+
+        $qb = $this->createQueryBuilder('self');
+        $expression = $qb->expr();
+
+        $qb
+            ->select(
+                'IDENTITY(self.mediaRelaySet) as mediaRelaySet'
+            )
+            ->where(
+                $expression->eq(
+                    'self.brand',
+                    $brandId,
+                )
+            );
+
+        $results = $qb->getQuery()->getArrayResult();
+
+        $response = [];
+        foreach ($results as $result) {
+            $response[] = (int) $result['mediaRelaySet'];
+        }
```

Comment:
Try to solve this casting and logic outside

Reply:
This is called from API, so I don't have a chance to work on the caller side.